javatests/arcs/core/entity/HandleManagerTestBase.kt

```diff
@@ -30,6 +29,8 @@ import kotlinx.coroutines.Deferred
 import kotlinx.coroutines.runBlocking
 import kotlinx.coroutines.test.runBlockingTest
 import org.junit.Test
+import java.lang.IllegalStateException
+import arcs.core.storage.Reference as StorageReference
```

Comment:
Are these unused? (the diff doesn't show them being used..., and IllegalStateException comes for free from kotlin's stdlib.